app/charts/shared/chart-helpers.tsx

```diff
@@ -437,10 +437,10 @@ const getBaseWideData = ({
   ) => Array<{ [key: string]: number }>;
 }): Array<Observation> => {
   const wideData = [];
-  const sortedDataGroupedByXEntries = [...dataGroupedByX.entries()].sort();
+  const dataGroupedByXEntries = [...dataGroupedByX.entries()];
```

Comment:
Is it sorted due to the sparql query ?

Reply:
It sorted due to calling `sortData` function in chart's props files, e.g. [here](https://github.com/visualize-admin/visualization-tool/blob/a3325d07e716beccf7224aed4fa76750dc09a074/app/charts/area/areas-state-props.ts#L80-L87)

Reply:
@ptbrowne I made the sorting explicitly required in places where it's needed via https://github.com/visualize-admin/visualization-tool/pull/1419/commits/2a8fc58cea5e9dd69e311d82fc9342091fb9c5fe, I think it also should cover the test that checks for sorting 🤞